server/models/Pet.js

```diff
@@ -13,17 +12,18 @@ class Pet {
     this.pet_type_id = pet_type_id || petTypeId
   }
 
-  static async findAllPets() {
+  static async findAll() {
     const queryData = { queryString: "SELECT * FROM adoptable_pets;" };
     const petData = await query(queryData);
     return petData.map((pet) => new this(pet));
   }
 
-  static async findPetById(id){
+  static async findById(id){
     const queryData = { queryString: "SELECT * FROM adoptable_pets WHERE id = $1;", values: [id] };
     const petData = await query(queryData);
     return new this(petData[0]);
   }
+  

```

Comment:
Where possible, let's try to only commit code to be merged if it's complete and working